src/Umbraco.Web.UI.Client/src/libs/controller-api/controller-host-element.element.ts

```diff
@@ -0,0 +1,9 @@
+import { UmbControllerHostElementMixin } from './controller-host-element.mixin.js';
+import { customElement } from '@umbraco-cms/backoffice/external/lit';
+
+/**
+ * A web-component to host umb controllers.
+ * This enables controllers to be added to the life cycle of this element.
+ */
+@customElement('umb-controller-host')
+export class UmbControllerHostElementElement extends UmbControllerHostElementMixin(HTMLElement) {}
```

Comment:
Any reason for the double "Element" ending?

Reply:
Yes, to because that was already taken by an interface... :-P

Reply:
But maybe it should be called Impelementation or Base or ?